packages/gitgraph-core/src/tag.ts

```diff
@@ -0,0 +1,37 @@
+import { TagStyle, CommitStyle, DEFAULT_FONT } from "./template";
+import { numberOptionOr } from "./utils";
+
+export { Tag };
+
+class Tag {
+  /**
+   * Name
+   */
+  public readonly name: string;
+  /**
+   * Style
+   */
+  public get style(): TagStyle {
```

Comment:
Using a getter to compute the `tag.style` at runtime. I find it a very nice alternative to computing everything in the constructor.